src/utils/__tests__/devfile-utils.spec.ts

```diff
@@ -1,4 +1,83 @@
-import { mockItems, getDevfileSamples } from '../devfile-utils';
+import { getDevfileSamples, DevfileSample } from '../devfile-utils';
+
+/**
+ * TODO: remove mock data for samples
+ */
```

Comment:
Tiny nit: The TODO text won't be needed anymore I guess

Reply:
Removed